src/components/workflow/WorkflowList.vue

```diff
@@ -234,6 +239,8 @@ export default {
     },
   },
   mounted() {
+    const self = this;
+    self.activeWorkflowSession = localStorage.getItem("activeWorkflowSession");
     this.getWorkflowList();
```

Comment:
We have amp-store set up for all global vars. See state.js. instead of using localStorage, can you add activeWorkflowSession there?

Reply:
The reason I've added activeWorkflowSession in local storage because if we open the workflow editor in one tab and try to open another workflow editor it should show edit buttons as disabled. If I keep the activeWorkflowSession in global vars, when we'll open 2 separate tabs for the same work the front end app will reset all the variables into it's initial values respectively!